Hazel/src/Hazel/ImGui/ImGuiConsole.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include "Hazel\Core.h"
+
+namespace Hazel {
+
+	class ImGuiConsole
+	{
+	public:
+		class Message
+		{
+		public:
+			enum class Level : int8_t
+			{
+				Invalid = -1,
+				Trace = 0,
+				Debug = 1,
+				Info = 2,
```

Comment:
Hey, Santa you should flip the debug and info enums so that Info is first and debug is second. This will make sure that your buttons and error levels function properly.